### ISSUE TITLE
Run MolGAN and Document Procedure 

### ISSUE THREAD

**Issue by szaman19** [szaman19/Materials-Search#4]
Install and run pytorch-geometric. Document the following: 

1.  Installation procedure. Generate a requirements.txt file to allow others to install quickly
2.  Run the file in: Moledule_Generation/MolGAN/main.py
3. Document input shapes and parameters for the model. 
    a. Generator input 
    b. Generator output 
    c. Discriminator input 
    d. Discriminator output 
4. Document loss function of the GAN. 

Create a readme.md file in Moledule_Generation/MolGAN for the writeup. Ask @szaman19 for any additional details on what to write.

**Comment by atenchen** [comment]
Okay, Got it.
